src/ctelnet.cpp

```diff
@@ -1308,11 +1308,14 @@ void cTelnet::setGMCPVariables(const QByteArray& msg)
         transcodedMsg = QString::fromUtf8(msg);
     }
 
-    QString var = transcodedMsg.section(QChar::Space, 0, 0);
-    QString arg = transcodedMsg.section(QChar::Space, 1);
-    if (arg.isEmpty()) {
-        var = transcodedMsg.section(QChar::LineFeed, 0, 0);
-        arg = transcodedMsg.section(QChar::LineFeed, 1);
+    QString packageMessage;
+    QString data;
+    packageMessage = transcodedMsg.section(QChar::Space, 0, 0);
```

Comment:
Why declare the local variable separately from assigning to it - I'd have thought it was less efficient though I'd lay reasonable odds that a decent compiler will spot it and convert this, the preceding two lines and the following one to:
```
QString packageMessage(transcodedMsg.section(QChar::Space, 0, 0));
QString data(transcodedMsg.section(QChar::Space, 1));
```
itself...!